llvm/utils/TableGen/DecoderEmitter.cpp

```diff
@@ -1048,90 +1064,150 @@ void DecoderEmitter::emitInstrLenTable(formatted_raw_ostream &OS,
 }
 
 void DecoderEmitter::emitPredicateFunction(formatted_raw_ostream &OS,
-                                           PredicateSet &Predicates,
-                                           indent Indent) const {
```

Comment:
Can this be done as a separate change?

Reply:
Started https://github.com/llvm/llvm-project/pull/148718 for this.